maxim/logger/llamaindex/utils.py

```
@@ -0,0 +1,101 @@
+    Parse the content of a list of LlamaIndex content blocks to a string.
+    """
+    text_content = []
+    for block in blocks:
+        if block.block_type == "text":
+            text_content.append(block.text)
+        elif block.block_type == "image":
+            if block.url:
+                if isinstance(block.url, str):
+                    image_url = block.url
+                else:
+                    image_url = str(block.url)
```

Ruff (0.12.2)

Use ternary operator `image_url = block.url if isinstance(block.url, str) else str(block.url)` instead of `if`-`else`-block

Replace `if`-`else`-block with `image_url = block.url if isinstance(block.url, str) else str(block.url)`

(SIM108)